src/logic/currencyValues/api/fetchCurrenciesRates.ts

```diff
@@ -9,11 +9,24 @@ const fetchCurrenciesRates = async (
 ): Promise<number> => {
   let rate = 0
   const url = `${getExchangeRatesUrl()}?base=${baseCurrency}&symbols=${targetCurrencyValue}`
+  const fallbackUrl = `${getExchangeRatesUrlFallback()}?currency=${baseCurrency}`
 
-  const result = await axios.get(url)
-  if (result && result.data) {
-    const { rates } = result.data
-    rate = rates[targetCurrencyValue] ? rates[targetCurrencyValue] : 0
+  try {
+    const result = await axios.get(url)
+    if (result && result.data) {
+      const { rates } = result.data
+      rate = rates[targetCurrencyValue] ? rates[targetCurrencyValue] : 0
+    }
+  } catch (error) {
+    try {
+      const result = await axios.get(fallbackUrl)
+      if (result && result.data) {
+        const { rates } = result.data?.data
```

Comment:
you already checked if result.data is defined, no need for optional chaining

Reply:
or you can move optional chaining to if statement

Reply:
> Is it me or this PR doesn't include anything related to new txs tab?

Sorry my bad, I just mixed the branches and merged `balances_enhacements` with this, now should be fixed. I will apply your comments to [that PR](https://github.com/gnosis/safe-react/pull/1122)